tests/test_claim.py

```diff
@@ -217,12 +213,26 @@ def test_generates_create_claim_with_partial_reverse_geocode(reverse_geocode_moc
 
 
 def test_generates_update_claim():
-    claim = _claim.generate_update()
+    claim = _claim.generate_update("hyphacoop")
     assert claim["vendor"] == "starlinglab"
 
 
 def test_generates_store_claim():
-    claim = _claim.generate_store("a-made-up-cid")
+    # Setup some organization-specific configuration
+    config.ORGANIZATION_CONFIG.config = {
+        "hyphacoop": {
+            "creative_work_author": [
+                {
+                    "@id": "https://twitter.com/somefakestuff",
+                    "@type": "Organization",
+                    "identifier": "https://example.com",
+                    "name": "hyphacoop",
+                }
+            ]
+        }
+    }
+
+    claim = _claim.generate_store("a-made-up-cid", "hyphacoop")
```

Comment:
Think these have to match bc of how Adobe's verifier renders the text.

```suggestion
                    "@id": "https://twitter.com/example",
                    "@type": "Organization",
                    "identifier": "https://example.com",
                    "name": "example",
                }
            ]
        }
    }

    claim = _claim.generate_store("a-made-up-cid", "example")
```